### PR TITLE
TizenRT support for SPI

### PR DESCRIPTION
**Pull request by kolipka**
Tested with osciloscope and Artik Sensor Board.

IoT.js-DCO-1.0-Signed-off-by: Konrad Lipner k.lipner@samsung.com

### CODE REVIEW

**Comment by yichoi** [comment]
@kolipka BTW, please add your test result on real target like https://samsung.github.io/iotjs-test-results/

**Comment by kolipka** [comment]
@yichoi I have that on my schedule, but first I have to make a way how to do it. Current TizenRT have missing ROM feature (https://github.com/Samsung/TizenRT/issues/118) and only way to actually put something on FS there is to embed scripts as a string to C file, and write on OS boot.

**Comment by yichoi** [comment]
@kolipka please discuss about this with @haesik or TizenRT team directly
we have an experience about putting ``js`` file to ARTIK05x/TizenRT at TDC demonstration

**Comment by yichoi** [comment]
@kolipka please rebase it to the master

File: src/platform/tizenrt/iotjs_module_spi-tizenrt.c [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      "maxSpeed: %d\n bitPerWord: %d\n loopback: %d",
+      _this->mode, _this->chip_select, _this->bit_order, _this->max_speed,
+      _this->bits_per_word, _this->loopback);
+
+  return true;
+}
+
+bool iotjs_spi_transfer(iotjs_spi_t* spi) {
+  IOTJS_VALIDATED_STRUCT_METHOD(iotjs_spi_t, spi);
+
```
Comment (LaszloLango) [inline src/platform/tizenrt/iotjs_module_spi-tizenrt.c]: Minor style issue: please use 2 empty lines before the function and only 1 inside it.

Comment (kolipka) [inline src/platform/tizenrt/iotjs_module_spi-tizenrt.c]: done

**Pull request merged**